pkg/querybuilder/time.go

```diff
@@ -0,0 +1,18 @@
+package querybuilder
+
+import "math"
+
+// ToNanoSecs takes epoch and returns it in ns
+func ToNanoSecs(epoch uint64) uint64 {
```

Comment:
lets add a basic test file atleast for happy flow ?

Reply:
I think a test file will be better with just the basic ones, i.e seconds, milliseconds, microseconds, and ns to ns.